### DISCUSSION TITLE
cypress showing an input as disabled

### DISCUSSION THREAD

**Discussion by PatoCodas** [cypress-io/cypress#30125] [Questions and Help]
I have a problem using cypress for automation.
As soon as I open a record editing page, and try to write something in an input, it tells me that the field is disabled, but when I open the same page manually, I can write normally.
I've already changed the elements I'm using, I've used different parameters, I've even tried navigating the whole page via tab.
Has anyone experienced this or knows what's going on?

**Comment by mah-noor1** [comment]
Could you please attach the reproducible code snippets here?
